Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -139,10 +139,22 @@ void HotkeyScheduler::Run()
 
   while (!m_stop_requested.IsSet())
   {
-    Common::SleepCurrentThread(1000 / 60);
+    const u64 freelook_update_freq = 200;
+    const u64 interval = SystemTimers::GetTicksPerSecond() / freelook_update_freq;
+    const u64 now = CoreTiming::GetTicks();
```

Comment:
CoreTiming::GetTicks() read might not be atomic on all architectures? Why don't you hook this inside the emulation thread give it depends on emulation timings? It's weird it's in the hotkeys thread if then it needs to depend on the current game speed.